### DISCUSSION TITLE
ipfHSVKey for Laue Groupe -1

### DISCUSSION THREAD

**Discussion by ThomasChauve** [mtex-toolbox/mtex#813] [Ask Anything] [answered]
Hello

My question is about the color coding in the Laue group -1. I don't know if I use the right naming for it. Therefore code and picture might be easier to explained my question.

If I run this :
`cs = crystalSymmetry('-1'); ipfKey = ipfHSVKey(cs); plot(ipfKey,'3d')`

![image](https://user-images.githubusercontent.com/9338747/109009406-cd8eb180-76ae-11eb-83d2-24d8d5c1dada.png)

I get a color coded sphere that show equivalent vector with the same color.

My probleme is that this colorcoding is discontinous. 

I also got this warning :

`Warning: Not a topological correct colormap! Please use the point group "1" `

Does that means that it mathematically impossible or that a correct way is not implemented in MTEX ?

Best
Thomas

**Comment by ralfHielscher** [comment, marked as answer]
It is mathematically impossible to have a jump free key for -1 that assigns different colors to different dirctions.

Ralf.

**Reply by ThomasChauve** [reply]
OK thanks for your quick answer
